identify-ref-leaks/ref-leak-identify.stp

```diff
@@ -0,0 +1,139 @@
+/*
+ * This is system tap script used to analyze reference leaks in glusterfs
+ * by probing referencing/dereferencing functions, putting a count on the
+ * number of times they are triggered, storing their traces if the count is 
+ * non-zero and printing them periodically.
+ */
+
+global count[600000]
+global traces_ref[80000]
+global traces_unref[80000]
+global ptr
+global round
+global i
+global j
+global status
+
+function dump()
+{
+        //delete all data structures
+        delete count
+        delete traces_ref
+	delete traces_unref
+	i = 0
+	j = 0
+}
+
+function aggregate()
+{
+        /* calculate net ref count */
+        printf("Net ref counts for each ptr : \n")
+        foreach (c in count) {
+       	        if (count[c] != 0) {
```

Comment:
This is a very awkward indention. Maybe you can correct that here and the other places where it is off.